Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -1206,10 +1206,10 @@ void LoadEventNexus::init() {
   setPropertyGroup("FilterMonByTimeStart", grp4);
   setPropertyGroup("FilterMonByTimeStop", grp4);
 
-  declareProperty("SpectrumMin", (int32_t)EMPTY_INT(), mustBePositive,
-                  "The number of the first spectrum to read.");
-  declareProperty("SpectrumMax", (int32_t)EMPTY_INT(), mustBePositive,
-                  "The number of the last spectrum to read.");
+  declareProperty("SpectrumMin", static_cast<int32_t>(EMPTY_INT()),
+                  mustBePositive, "The number of the first spectrum to read.");
+  declareProperty("SpectrumMax", static_cast<int32_t>(EMPTY_INT()),
+                  mustBePositive, "The number of the last spectrum to read.");
```

Comment:
Not sure I like this. Does it rely on the return value of EMPTY_INT (INT_MAX) being the equal to INT32_MAX? But I am not sure what to do about it.